routing/pathfind.go

```diff
@@ -395,7 +384,7 @@ func newRoute(amtToSend, feeLimit lnwire.MilliSatoshi, sourceVertex Vertex,
 
 	// The total amount required for this route will be the value the
 	// source extends to the first hop in the route.
-	route.TotalAmount = runningAmt
+	route.TotalAmount = route.Hops[0].AmtToForward + route.Hops[0].Fee
```

Comment:
Comment should be updated to explain the calculation.

Reply:
Updated

Reply:
Nice. I would also add a note that the source node (the sender of the payment) _is not_ part of the hops here.